exercises/concept/troll-the-trolls/.docs/instructions.md

```diff
@@ -0,0 +1,90 @@
+# Instructions
+
+Your friend Helma made a small online game that rapidly gained popularity.
+It is called _HellMath_.
+The small community attracted some trolls who make the game and the forums pretty unpleasant.
+Helma has asked you to work on a new permission system to separate "the wheat from the chaff". 
+
+The forum supports three different actions:
+
+- Read
+- Write
+- Remove
+
+There are four types of accounts, each with different default permissions:
+
+- Guests: can read posts.
+- Users and Trolls: can read and write posts.
+- Moderators: can read, write and remove posts, they have all the permissions.
+
+Helma has noticed that it is no use to ban troll accounts.
+Her strategy is to give them the illusion that their time is "well invested", but their posts are only shown to other trolls.
+For anything that requires a priority, trolls get the least places.
+When they enter a game, the pool of available players is also limited to other trolls.
+
+## 1. Set up user types and permissions.
+
+First, define an `AccountStatus` enumeration to represent the three account types: `troll`, `guest`, `user`, and `mod`.
+
+Next, define an `Action` enumeration to represent the three permission types: `read`, `write`, `remove`.
```

Comment:
read, write, remove -> read, write, and remove
or remove the "and" from the previous line.